main.js

```diff
@@ -308,7 +319,27 @@ if (shouldQuit) {
 
 app.on('ready', () => {
   if ((process.env.NODE_ENV === 'production') || process.env.SLOBS_FORCE_AUTO_UPDATE) {
-    (new Updater(startApp)).run();
+    const updateInfo = {
+      baseUrl: 'https://d1g6eog1uhe0xm.cloudfront.net',
+      version: pjson.version,
+      exec: process.argv,
+      cwd: process.cwd(),
+      waitPids: [ process.pid ],
+      appDir: path.dirname(app.getPath('exe')),
+      tempDir: path.join(app.getPath('temp'), 'slobs-updater'),
+      cacheDir: app.getPath('userData'),
+      versionFileName: `${releaseChannel}.json`
+    };
+
+    console.log(updateInfo);
```

Comment:
Use the `log` method now. It combines the logs from all renderer and main process into one file